src/axom/mir/examples/concentric_circles/mir_concentric_circles.cpp

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "axom/config.hpp"
+#include "axom/core.hpp"  // for axom macros
+#include "axom/slic.hpp"
+#include "axom/mir.hpp"  // for Mir classes & functions
+#include "runMIR.hpp"
+
+#include <conduit.hpp>
+#include <conduit_relay_io_blueprint.hpp>
+
+#include <string>
+
+// namespace aliases
+namespace mir = axom::mir;
+namespace bputils = axom::mir::utilities::blueprint;
+
+using RuntimePolicy = axom::runtime_policy::Policy;
+
+//--------------------------------------------------------------------------------
+void conduit_debug_err_handler(const std::string &s1, const std::string &s2, int i1)
+{
+  SLIC_ERROR(
+    axom::fmt::format("Error from Conduit: s1={}, s2={}, i1={}", s1, s2, i1));
+  // This is on purpose.
+  while(1)
+    ;
+}
+
+//--------------------------------------------------------------------------------
+void printNode(const conduit::Node &n)
+{
+  conduit::Node options;
+  options["num_children_threshold"] = 10000;
+  options["num_elements_threshold"] = 10000;
+  n.to_summary_string_stream(std::cout, options);
+}
+
+//--------------------------------------------------------------------------------
+int runMIR(RuntimePolicy policy,
+           int gridSize,
+           int numCircles,
+           const std::string &outputFilePath)
+{
+  // Initialize a mesh for testing MIR
+  auto timer = axom::utilities::Timer(true);
+  mir::MeshTester tester;
+  conduit::Node mesh;
+  {
+    AXOM_ANNOTATE_SCOPE("generate");
+    tester.initTestCaseFive(gridSize, numCircles, mesh);
+    // printNode(mesh);
+  }
```

Comment:
As we discussed offline -- it would be nice to replace this `mir::MeshTester` volume fraction generation with blueprint meshes generated by our shaping example in the future (e.g. a follow-up branch).